portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,27 +20,39 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
-import java.util.Arrays;
 import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  ArrayList<String> messages = new ArrayList<String>( 
-    Arrays.asList("msg1", 
-                  "msg2", 
-                  "msg3")); 
+  class Comment {
```

Comment:
`private static class` (failing to include `static` here means you are creating one "class type" per instance of DataServlet, which is really not what you want).